recipes/ldns/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest C)
+
+find_package(PkgConfig)
+pkg_check_modules(LDNS REQUIRED "ldns")
```

Comment:
there likely needs to be some logic in the `package_info()` to setup the components